cabby/geo/geo_computation.py

```diff
@@ -38,16 +45,21 @@
 def main(argv):
     del argv  # Unused.
 
-    # Compute graph over Manhattan.
+    print("Obtaining graph for Manhattan.")
```

Comment:
nit: use a logger instead?

Reply:
Added a TODO for tzufgoogle@ to do that in another PR.